testing/run-scripts/run_monitor.sh

```diff
@@ -0,0 +1,18 @@
+# Boot monitoring servers and metrics.
```

Comment:
So, standard bash stuff:
 * `#/bin/bash` to open
 * you probably want `set -e`

Use `run_cloud.sh` as a template.